geane/FairGeanePro.cxx

```diff
@@ -808,10 +850,10 @@ void FairGeanePro::Track2ToLine( TVector3 X1,  TVector3 X2,  TVector3 w1,
   TVector3 x21, x32, w21;
   TVector3 xw1, xw2;
 
-  Double_t a1, b1, c1, d1, e1, t1, s1;
-  Double_t Delta1;
+  double a1, b1, c1, d1, e1, t1, s1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 's1' can be reduced.](https://app.codacy.com/app/FairRootGroup/FairRoot/pullRequest?prid=4139720)